src/main/java/com/exasol/releaserobot/report/Report.java

```diff
@@ -1,9 +1,27 @@
 package com.exasol.releaserobot.report;
 
+import java.util.List;
+
+import com.exasol.releaserobot.report.ReportImpl.ReportName;
+
 /**
  * Represents a report.
  */
 public interface Report {
+    /**
+     * Add a new result.
+     *
+     * @param result result
+     */
+    public void addResult(Result result);
+
+    /**
+     * Add a list of results.
+     *
+     * @param results list of results
+     */
+    public void addResults(List<? extends Result> results);
```

Comment:
I think this method can go now, given that we it's used only in the reportImpl class